src/example-types/metricGroups/pivot.js

```diff
@@ -31,15 +31,82 @@ let aggsForValues = (node, schema) =>
     }))
   )(node)
 
-let maybeWrapWithFilterAgg = ({ query, filters, skipFilters, aggName }) =>
-  _.isEmpty(filters) && _.isEmpty(skipFilters)
+let getKey = x => x.keyAsString || x.key
+
+let drilldownLookup = (type, path, results) => {
```

Comment:
I'm not sure I follow exactly what this method is trying to do. Unit tests would be helpful (if nothing else but to aid in reviewing)

Reply:
This method returns sub-results for a given drilldown. Drilldown is using both `key` and `keyAsString` so we can't just use Tree.lookup